run_tests.sh

```diff
@@ -61,7 +61,6 @@ fi
 
 # Environment initialization
 if [[ "$package_type" == conda || "$(uname)" == Darwin ]]; then
-    EXTRA_CONDA_FLAGS="-c=conda-forge"
```

Comment:
I think we still need to keep it for Darwin and Python-3.9, aren't we?

Reply:
Just tried the following and it appears to work fine,

```
❯ conda create -y -p $(pwd) -c pytorch pytorch python=3.9
```